Don't fail on empty staple files

manuelm opened the pull request:
HAProxy intentionally allows empty stable files and treat them as non-existent. In fact creating an empty stable allows to set the stable at runtime which isn't possible if it's omitted beforehand. Parsing an empty file won't work, so handle it as non existent as well.

SnijderC commented on ocspd/core/certmodel.py:
```diff
@@ -103,6 +103,11 @@ def recycle_staple(self, minimum_validity):
             return False
 
+        # from haproxy docs: [...] The content of this file is optional [...]
+        if len(staple) == 0:
```
`staple` is not yet defined at this point, did you intend to add this at line ~120?

manuelm replied:
erm sorry. shouldn't use the webeditor…
lines are meant to go to 115 just before the `staple = OCSPResponseParser(staple)`. will fix this

manuelm committed:
Don't fail on empty staple files

HAProxy intentionally allows empty stable files and treat
them as non-existent. In fact creating an empty stable
allows to set the stable at runtime which isn't possible if
it's omitted beforehand. Parsing an empty file won't work,
so handle it as non existent as well.

SnijderC committed:
Add a more usefull comment

in case someone will wonder what is going on here in the future.

SnijderC commented:
Thanks for your input!
